### PR TITLE
refactor: exhibitors tab

### PR DESCRIPTION
**Pull request by tr-ann**
The purpose of the PR is **rename** existing "Exhibitors" tab and route to "Booths" to avoid naming collision when add new "Exhibitors A-Z" tab.

Made some changes to improve the code

### CODE REVIEW

File: src/v2/Apps/Fair/FairApp.tsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ const FairApp: React.FC<FairAppProps> = ({ children, fair }) => {
           onClick={trackTabData(
             `${fairHref}/booths`,
             "Booths",
-            ContextModule.exhibitorsTab
+            "boothsTab" as ContextModule
```
Comment (dblandin) [inline src/v2/Apps/Fair/FairApp.tsx:83]: **suggestion / non-blocking**: Let's make sure to ticket this new context so we can swap `ContextModule.boothsTab` in here.

cc/ @louislecluse FYI ☝🏼

**Review by dblandin — APPROVED** [review]
LGTM! A couple minor comments. Feel free to merge when ready!

File: src/v2/Apps/Fair/FairApp.tsx (hunk 1 above)
Comment (tr-ann) [reply src/v2/Apps/Fair/FairApp.tsx:83]: @dblandin should I do something with it? Or leave it as it is?

Comment (dblandin) [reply src/v2/Apps/Fair/FairApp.tsx:83]: You can leave it as is for this PR! I'll follow up with another Jira ticket.

**Pull request merged**